docs/en/Storage.md

```diff
@@ -1,132 +1,261 @@
-# Storage
+# Storage Types
```

Comment:
Silly point, but at the moment there is only this single `h1` header, which makes the table-of-contents tree a bit weird. Maybe just move `Storage Types` to the title or ditch it? Then the table-of-contents tree will make more sense.

Reply:
totally right.  `Storage Types` feels more like an overview now anyway, so I'm changing the first to `Overview` then shifting everything else up a level (so `Overview`, `Disks`, and `Buckets` are all top level)